docs/advanced-topics/networkobject-parenting.md

```diff
@@ -4,22 +4,27 @@ title:  NetworkObject Parenting
 description: A `NetworkObject` parenting solution within Netcode for GameObjects (Netcode) to help developers with synchronizing transform parent-child relationships of `NetworkObjects`.
 
 ---
-
-:::important Opt-OUT
-This feature is behind a bool flag that can be toggled on the `NetworkObject` inspector UI. It will be enabled by default but you can opt-out from it if you want to implement your own solution
+:::note Optional Auto Synchronized Parenting
+The Auto Object Parent Sync property of NetworkObject, enabled by default, allows you to disable automatic parent change synchronziation in the event you want to implement your own parenting solution for one or more NetworkObjects.
 :::
 
- [`MonoBehaviour.OnTransformParentChanged()`](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnTransformParentChanged.html) under `NetworkObject`  is utilized to catch `transform.parent` changes.
-
-Three additional state variables are stored in `NetworkObject`:
-
-```csharp
-bool m_IsReparented; // did parent change compared to initial scene hierarchy?
-ulong? m_LatestParent; // who (NetworkObjectId) is our latest (current) parent if we changed our parent?
-Transform m_CachedParent; // who (Transform) was our previously assigned parent?
-```
+## NetworkObject Parenting Rules
 
-`NetworkBehaviour` includes a virtual method you can override to be notified when a `NetworkObject`'s parent has changed:
+### Overview
+- Setting the parent of a child's `Transform` directly (i.e. `transform.parent = childTransform;`) will always use the default `WorldPositionStays` value of `true`.
```

Comment:
Yes.
```
// WorldPositionStays defaults to true, clients are not synchronized but since this is the default it is the same as if it was synchronized
NetworkObject.transform.parent = parent.transform;  
// WorldPoisitonStays defaults to true, clients are not synchronized but since this is the default it is the same as if it was synchronized
NetworkObject.transform.SetParent(parent.transform);  
// WorldPositionStays will be false, but will not synchronize clients
NetworkObject.transform.SetParent(parent.transform, false); 
// WorldPositionStays will always be true (like the transform.SetParent), but NGO will synchronize this with clients
NetworkObject.TrySetParent(parent); 
// WorldPositionStays is false and NGO will synchronize this with clients
NetworkObject.TrySetParent(parent, false); 
```